### ISSUE TITLE
WebGL renderer doesn't support hidden modifier

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#87918]
From https://github.com/xtermjs/xterm.js/issues/2596

## Details
- Browser and browser version: Code Insiders 2019-12-03
- OS version: Windows 10

### Steps to reproduce

Run this test script, notice the "hidden" text is not actually hidden when using the WebGL renderer:

```js
const chalk = require('chalk');

const modifiers = ["reset", "bold", "dim", "italic", "underline", "inverse", "hidden", "strikethrough", "visible"];
const colors = ["black", "red", "green", "yellow", "blue", "magenta", "cyan", "white", "blackBright", "redBright", "greenBright", "yellowBright", "blueBright", "magentaBright", "cyanBright", "whiteBright"];

const colWidth = 20;

for (const modifier of modifiers) {
    process.stdout.write(chalk[modifier](modifier.padEnd(colWidth)));
}

process.stdout.write('\r\n');
for (const color of colors) {
    for (const modifier of modifiers) {
        process.stdout.write(chalk[modifier][color](color.padEnd(colWidth)));
    }
    process.stdout.write('\r\n');
}
```

https://github.com/microsoft/vscode/issues/85971

**Comment by aeschli** [comment]
Version: 1.42.0-insider (user setup) 
8d4cdbaa8d64514a5eb3b186b9a31b66cd8a9753
Date: 2020-01-31T08:56:44.663Z
I ran the sample above in the integrated terminal.
Also I tried all the options of `"terminal.integrated.rendererType"`, including "experimentalWebgl"

Hidden never seem to be hidden.
![image](https://user-images.githubusercontent.com/6461412/73547833-06267200-4440-11ea-9ebe-8b19225f356f.png)

Sorry If I misunderstood the issue.
